src/main/java/dev/cdevents/CDEventTypes.java

```diff
@@ -231,6 +231,7 @@ private static CloudEventBuilder buildCloudEvent(
                 .withId(UUID.randomUUID().toString())
                 .withSource(URI.create("cdevents-sdk-java"))
                 .withType(eventType)
+                .withDataContentType("application/json; charset=UTF-8")

```

Comment:
Would it make sense to add a test that asserts all CDEvents created with the builder have a content type set?

Reply:
"Soon" there will be json schemas published for the CDEvents protocol, se https://github.com/cdevents/spec/pull/61. When that is done, and when the Java SDK is lifted to the latest CDEvents spec version, we should have tests in all SDKs that verify that the produced events validate towards the json schemas for the protocol. If/when we have that, is there still a reason to assert the content type?

Reply:
Excuse my ignorance, I wasn't aware of that pull request. To answer your question: probably not, if the the json schemas enforce the content type to be specified.

Reply:
Oups, now you must excuse me :) I was too quick in my reply previously. The data content type will not be part of the protocol itself, and therefore not part of the protocol schema. There will be "supported bindings", of which json is the main one, but that is on higher level than the schema itself.
So, it might be wise to actually assert that the content type is set within the Java SDK already now.